config/initializers/adobe.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Get credentials from database
+destination = Destination.adobe_campaigns.first
+if destination&.adobe_api_key.present? && destination&.adobe_api_secret.present?
```

Comment:
Note that existing columns `access_key_id` and `access_key_secret` should be referenced instead of `adobe_api_key` and `adobe_api_secret`, respectively.

Reply:
@hlbraddock got it, I didn't get that part originally.